Samples/iOS-SwiftUI/iOS-SwiftUI.xcodeproj/project.pbxproj

```diff
@@ -19,6 +19,20 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXContainerItemProxy section */
+		0A94157D28F6B893006A5DD1 /* PBXContainerItemProxy */ = {
```

Comment:
I don't know why I got these changes in the project file for iOS-SwiftUI. I did open that project recently instead of iOS-Swift, I guess just opening it must've triggered something in Xcode?